aws/policy/security-services.yaml

```diff
@@ -87,6 +87,48 @@ Statement:
       - waf:UpdateSqlInjectionMatchSet
       - waf:UpdateWebACL
       - waf:UpdateXssMatchSet
+      - wafv2:ListRuleGroups
```

Comment:
This looks like every action for wafv2 except UpdateWebACL - are these all needed for the integration tests?

Reply:
The PRs covers list, get, create, update and delete of
  * web_acl
  * rule_group
  * ip_set

plus associate and disassociate a web acl to a resource.   
So it is nearly everything. What can be removed is
  * List, Get, Create, Update and Delete of RegexPatternSet (_not implemented yet_)
  * Tag/Untag resources (_you can tag resources on create, but tags are not supported currently, nor via boto3, nor via cli, nor via web console_)
  * Put, Get, Delete PermissionPolicy. (_I guess it is not used, but I'm not 100% sure if it is used automagically in the background_)